src/MassTransit/Pipeline/Filters/ConsumerMessageFilter.cs

```diff
@@ -48,8 +48,8 @@ async Task IFilter<ConsumeContext<TMessage>>.Send(ConsumeContext<TMessage> conte
                 MessageType = TypeMetadataCache<TMessage>.ShortName
             });
 
-            activity?.AddTag(DiagnosticHeaders.Kind, DiagnosticHeaders.SpanKind.Consumer);
-            activity?.AddTag(DiagnosticHeaders.PeerAddress, TypeMetadataCache<TConsumer>.ShortName);
+            activity?.AddTag(DiagnosticHeaders.ServiceKind, DiagnosticHeaders.Kind.Consumer);
+            activity?.AddTag(DiagnosticHeaders.DestinationAddress, TypeMetadataCache<TConsumer>.ShortName);

```

Comment:
How is the consumer type a peer.address? Seems wrong, should be the input queue address.

Reply:
My thinking was to distinguish between multiple consumers that might be on the same receive endpoint.

Reply:
Note also that the Receive traces will already have the input address